render/egl.c

```diff
@@ -366,6 +382,72 @@ struct wlr_egl *wlr_egl_create_with_drm_fd(int drm_fd) {
 			wlr_log(WLR_DEBUG, "Obtained high priority context");
 		}
 	}
+	return true;
+}
+
+struct wlr_egl *wlr_egl_create_with_drm_fd(int drm_fd) {
+	struct wlr_egl *egl = wlr_egl_alloc();
+	if(egl == NULL)
```

Comment:
Style: always use braces, even for single-line conditions. Applies to the whole PR.

Reply:
Style: missing space between `if` and `(`. Applies to the whole PR.